config/locales/views/documents.fr.yml

```diff
@@ -0,0 +1,6 @@
+fr:
+  documents:
+    file_name: Nom du document
```

Comment:
It doesn't seem that this I8n key is actually used, as you use instead the lookup `'activerecord.attributes.document.filename'`. Also, the `en` and `fr` keys for this value are different.
If that's an attribute translation that you want, you can use `Document.human_attribute_name(:file_name)`